varnish/ci/Dockerfile_varnish5

```diff
@@ -0,0 +1,11 @@
+from debian:stretch
+
+RUN apt-get update -y \
+    && apt-get install -y apt-transport-https \
+    && apt-get install -y \
+        varnish \
```

Comment:
You should pin this to the version you're testing at, debian might update it out from under you otherwise